openownership/data-standard
# Docs restructure

# Overview

**What does this pull request do?**

- Restructures the 'Schema' section of the docs (now called 'Data Standard').
- Edits the docs conf.py file to change 0.3 -> 0.4

Note: the navigation at the top of the page still says 'Data schema' (not Data Standard) and will not take you to the right page. You will need to edit the url from 'schema' to 'standard'. The link is hardcoded in the [theme repo](https://github.com/openownership/data-standard-sphinx-theme), and a PR has been submitted to fix this.

**How can a reviewer test or examine your changes?**

Build the docs. Navigate around. Check that the changes are in line with the [plans](https://docs.google.com/spreadsheets/d/1gaARj0p7UShhDcHP83RFoRXPSstM8Zc4al5UZRA6AJQ/edit?pli=1#gid=1592918667)


**Who is best placed to review it?**

@odscrachel - Kathryn and I both worked on this, so your eyes on it would be good!


Closes #534

## Translations

n/a

## Documentation & Release
